inc/class-book.php

```diff
@@ -544,6 +551,52 @@ static function getSubsections( $id ) {
 		return $output;
 	}
 
+	/**
+	 * Returns an array of front matter, chapters, and back matter which contain subsections.
+	 *
+	 * @param array $book_structure The book structure from getBookStructure()
+	 * @return array The subsections, grouped by parent post type
+	 */
+	static function getAllSubsections( $book_structure ) {
+		if ( Export::isParsingSubsections() ) {
```

Comment:
This is the wrong time to notice this since this dates back to 2015 and the change was made to match coding standards (da87d42) but with several new changes related to cache stampedes, this feels like a weird function name?

Reply:
Agreed…